pkgs/development/libraries/science/math/liblapack/default.nix

```diff
@@ -33,6 +36,26 @@ stdenv.mkDerivation {
 
   doCheck = true;
 
+  # Some CBLAS related tests fail on Darwin:
+  #  14 - CBLAS-xscblat2 (Failed)
+  #  15 - CBLAS-xscblat3 (Failed)
+  #  17 - CBLAS-xdcblat2 (Failed)
+  #  18 - CBLAS-xdcblat3 (Failed)
+  #  20 - CBLAS-xccblat2 (Failed)
+  #  21 - CBLAS-xccblat3 (Failed)
+  #  23 - CBLAS-xzcblat2 (Failed)
+  #  24 - CBLAS-xzcblat3 (Failed)
+  #
+  # Upstream issue to track:
+  # * https://github.com/Reference-LAPACK/lapack/issues/440
+  ctestArgs = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) "-E '^(CBLAS-(x[sdcz]cblat[23]))$'";
```

Comment:
```suggestion
  ctestArgs = lib.optionalString stdenv.isDarwin "-E '^(CBLAS-(x[sdcz]cblat[23]))$'";
```
Also fail on x86_64

Reply:
Can confirm. This is needed on staging-next (https://github.com/NixOS/nixpkgs/pull/115435)

```
93% tests passed, 8 tests failed out of 120

Total Test time (real) =   1.46 sec

The following tests FAILED:
         14 - CBLAS-xscblat2 (Failed)
         15 - CBLAS-xscblat3 (Failed)
         17 - CBLAS-xdcblat2 (Failed)
         18 - CBLAS-xdcblat3 (Failed)
         20 - CBLAS-xccblat2 (Failed)
         21 - CBLAS-xccblat3 (Failed)
         23 - CBLAS-xzcblat2 (Failed)
         24 - CBLAS-xzcblat3 (Failed)
```

Reply:
Oh, missed this

Reply:
Still need to apply for x86_64-darwin

Reply:
Are we ok with just pushing this suggestion to staging-next as a fixup commit then?

Reply:
Here's the fix targeting `staging-next`: #116779.

Reply:
Done